scripts/build_single.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# for printf
+export LC_ALL=C
```

Comment:
```suggestion
export LC_NUMERIC=C
```
I think `LC_NUMERIC` would be more appropriate. By setting `LC_CTYPE=C` (which `LC_ALL=C` implicitly does), you tell the system to not expect UTF-8.

Reply:
i tried that first, doesn't work

Reply:
Ah, it's probably because `LC_NUMERIC` is being set to `en_US.UTF-8` below. You can probably edit that to `C` to fix the issue.

Reply:
![image](https://user-images.githubusercontent.com/6654924/153727633-8eb93762-d4a9-45a2-b1a0-45f89e5753d1.png)


The below code was maybe an earlier try to change it, but it doesnt seem to work for me anymore

Reply:
`env | grep -E '(LC|LANG)'`, please.

Reply:
```
LANGUAGE=
LANG=de_DE.UTF-8
LC_ALL=de_DE.UTF-8
```

Reply:
The problem is that you have set `LC_ALL=de_DE.UTF-8`. You shouldn't do that, it's an override. Everything defaults to `LANG`.

Reply:
If you want to fix the script to work in this situation, I'd let it unset `LC_ALL`, but I think that'd also be confusing.

Reply:
i didn't do anything

Reply:
now it should only affect the printf command directly